UnityProject/Assets/Scripts/Core/Input System/InteractionV2/TileInteraction/TableInteractionClimb.cs

```diff
@@ -56,54 +58,84 @@ public override bool WillInteract(TileApply interaction, NetworkSide side)
 
 	public override void ServerPerformInteraction(TileApply interaction)
 	{
+		if (interaction.UsedObject == null || interaction.UsedObject.TryGetComponent(out UniversalObjectPhysics objectPhysics) == false)
+		{
+			return;
+		}
 		if (!interaction.UsedObject.RegisterTile().Matrix.IsPassableAtOneMatrixOneTile(interaction.TargetCellPos, true, true, null, excludeTiles))
 		{
 			return;
 		}
+		StartClimbing(true, interaction.PerformerPlayerScript,
```

Comment:
what did you change about this function since the git compare has just brutalised it

Reply:
I moved it into its own function so I can call it from the CilmbTable strat

Reply:
cool